hw/ip/aes/dv/cov/refines/aes_common_assert.vRefine

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
it's weird that these assert coverage should be hit. I see most of them are hit in those VCS blocks. Please don't get this in, let me investigate a bit more.

Reply:
ok I will investigate a bit also!

Reply:
This is probably because we have to enable assert coverage collection in the CSR test build. I made an update for VCS at #11482.
Can you do the same for Xcelium? Then we can remove this file.

Reply:
fixed